src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -704,6 +704,7 @@ private TypeSymbol BindTupleType(TupleTypeSyntax syntax, BindingDiagnosticBag di
                     hasExplicitNames = true;
                     CheckTupleMemberName(name, i, nameToken, diagnostics, uniqueFieldNames);
                     locations.Add(nameToken.GetLocation());
+                    ReportFieldOrValueContextualKeywordConflictIfAny(argumentSyntax, nameToken, diagnostics);
```

Comment:
>ReportFieldOrValueContextualKeywordConflictIfAny [](http://example.com/codeflow?start=20&length=48)

It looks like there is another call site of ```CheckTupleMemberName``` helper that is not benefiting from a similar check. Should it be? If so, consider moving the check into ```CheckTupleMemberName```.

Reply:
The other caller of `CheckTupleMemberName()` is `ExtractTupleElementNames()` which extracts the names from the element expressions. Those names will be checked when the element expressions are bound.